### DISCUSSION TITLE
有几个配置参数没看明白

### DISCUSSION THREAD

**Discussion by jkfujr** [go-olive/olive#60] [Q&A] [answered]
这三
`SnapRestSeconds`
`SplitRestSeconds`
`CommanderPoolSize`

**Comment by NormanMises** [comment]
同 这好像会影响监控器和切分文件的时间间隔，但我不知道设置多少才合适

**Comment by luxcgo** [comment, marked as answer]
* SnapRestSeconds
    * 直播间状态查询间隔时间（秒）
* SplitRestSeconds
    * 文件是否满足切割条件检测间隔时间（秒）
* CommanderPoolSize
    * 直播间录播结束后执行命令的并发执行的个数
